package/network/ipv6/ds-lite/Makefile

```diff
@@ -17,13 +17,13 @@ define Package/ds-lite
   SECTION:=net
   CATEGORY:=Network
   DEPENDS:=@IPV6 +kmod-ip6-tunnel +resolveip
-  TITLE:=Dual-Stack Lite (DS-Lite) configuration support
-  MAINTAINER:=Steven Barth <steven@midlink.org>
+  TITLE:=IPv4 over IPv6 (RFC2473 and DS-Lite) configuration support
+  MAINTAINER:=Arayuki Mago <ms@missing233.com>
```

Comment:
maintainer takeover should happen in a separated commit.

Reply:
It has now been reverted back to the previous maintainer.